notebooks/EOPF_Tutorials/EOPF_SAFE_ZARR_COMPARE_METADATA_SENTINEL2_L2A.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        "/mnt/CEPH_PROJECTS/test/S2B_MSIL2A_20250206T101109_N0511_R022_T32TPS_20250206T122414.zarr",
</pre><p>Please use the data provided by EODC on the object storage. Let us know if something doesn't work as expected using that instead of the local Zarr.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EOPF-Sample-Service/eopf-sample-notebooks/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>